unstructured/metrics/evaluate.py

```diff
@@ -187,3 +177,46 @@ def measure_element_type_accuracy(
     _write_to_file(export_dir, "all-docs-element-type-frequency.tsv", df)
     _write_to_file(export_dir, "aggregate-scores-element-type.tsv", agg_df)
     _display(agg_df)
+
+
+def group_text_extraction_accuracy(
+    grouping: str, data_input: Union[pd.DataFrame, str], export_dir: str
+) -> None:
+    """
+    Aggregates accuracy and missing metrics by 'doctype' or 'connector', exporting to TSV.
+
+    Args:
+        grouping (str): Grouping category ('doctype' or 'connector').
+        data_input (Union[pd.DataFrame, str]): DataFrame or path to a CSV/TSV file.
+        export_dir (str): Directory for the exported TSV file.
+    """
```

Comment:
Docstrings should be [PEP 257](https://peps.python.org/pep-0257/) compliant, like this in this case:
```python
    """Aggregates accuracy and missing metrics by 'doctype' or 'connector', exporting to TSV.

    Args:
        grouping (str): Grouping category ('doctype' or 'connector').
        data_input (Union[pd.DataFrame, str]): DataFrame or path to a CSV/TSV file.
        export_dir (str): Directory for the exported TSV file.
    """
```

I encourage you to read the PEP, but the basic idea is:
- first line begins immediately after the """ and extends for no more than one line.
- second line is blank
- remaining lines as needed
- closing """ is on separate line, aligned at same indent level as first """

If only a single line docstring is needed it must fit on one line, like:

```python
    """Aggregate accuracy and missing metrics by 'doctype' or 'connector'."""
```

Reply:
Thank you!